.github/workflows/ci_pipeline.yml

```diff
@@ -42,8 +42,9 @@ jobs:
 
     strategy:
       matrix:
-        python: ['3.8', '3.9', '3.10']
         env: ['base', 'fenics', 'mpi4py', 'petsc']
+        # python: ['3.8', '3.9', '3.10']
+        python: ['3.10']
```

Comment:
Don't forget to revert this once we're ready to merge.

Reply:
Reverted in eebbdd3b89a71f3c3063efccfab926abde748fea